book/src/frequently_asked_questions.md

```diff
@@ -6,14 +6,32 @@
 
 You may have noticed that Kanidm requires you to configure TLS in your container or server install.
 
-We are a secure-by-design rather than secure-by-configuration system, so TLS for all connections is
-considered mandatory and a default rather than an optional feature you add later.
+One of the fundamental goals of the project is a secure-by-design rather than
+secure-by-configuration system, so TLS for all connections is mandatory. It is not an optional
+feature you add later.
 
-### Can Kanidm work without TLS?
+> [!NOTE]
+>
+> Please respect our decision. We've had this discussion. A lot.
+>
+> You're at the FAQ because you have questions, and this is the final answer on the matter.
+>
+> If this means you won't use Kanidm, thank you for your consideration.
```

Comment:
This is a bit harsh, maybe just reword to:

```
> Please respect the maintainers decision on TLS-by-default. 
```

Alternately a new segment

```
### I think TLS should be opt-out for my use case

We previously did allow this. It created a lot of support issues, confusing configurations and headaches. After a string of problems, we had an extensive discussion about the role of Kanidm and decided that both in the interest of secure defaults *and* single-config for uses, that TLS should be required between all components. 
```

Reply:
Yeah,  didn't want to allow the "but you did it in the past" hole to be opened up.